Buddies/Controls/FilterSearchBar.swift

```diff
@@ -47,87 +53,200 @@ class FilterSearchBar : UISearchBar, UISearchBarDelegate {
         self.delegate = self
         
         // Create the filter button
-        renderFilterButton()
+        let bttn = makeFilterButton(
+            doing: #selector(self.onFilterTapped),
+            rounding: [.layerMaxXMaxYCorner, .layerMaxXMinYCorner])
+        
+        addSubview(bttn)
+        
+        // Size/Position
+        NSLayoutConstraint.activate([
+            bttn.rightAnchor.constraint(equalTo: rightAnchor, constant: -10),
```

Comment:
Any reason why these are negative?

Reply:
I don't really know but they overlapped when it was positive and they looked correct when it was this direction.

Reply:
cool